learning_journal/test/tests.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+
+from pyramid import testing
+
+
+def test_detail():
+    from learning_journal.views import detail
+    request = testing.DummyRequest()
+    info = detail(request)
+    # import pdb; pdb.set_trace()
+    assert 'title' in info['entries'][0]
```

Comment:
Your detail view probably should only be returning a single entry, not a list of them.